num2words/lang_PT.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2003, Taro Ogawa.  All Rights Reserved.
+# Copyright (c) 2013, Savoir-faire Linux inc.  All Rights Reserved.
+
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301 USA
+
+from __future__ import division, unicode_literals
+
+import re
+
+from .lang_EU import Num2Word_EU
+
+
+class Num2Word_PT(Num2Word_EU):
+    CURRENCY_FORMS = {
+        'AUD': (('dólar', 'dólares'), ('cêntimo', 'cêntimos')),
+        'CAD': (('dólar', 'dólares'), ('cêntimo', 'cêntimos')),
+        'EUR': (('euro', 'euros'), ('cêntimo', 'cêntimos')),
+        'GBP': (('libra', 'libras'), ('centavo', 'centavos')),
+        'USD': (('dólar', 'dólares'), ('cêntimo', 'cêntimos')),
+    }
+
+    GIGA_SUFFIX = None
+    MEGA_SUFFIX = "ilião"
+
+    def set_high_numwords(self, high):
+        cap = 3 + 6 * len(high)
+
+        for word, n in zip(high, range(cap, 3, -6)):
+            if self.GIGA_SUFFIX:
+                self.cards[10 ** n] = word + self.GIGA_SUFFIX
+
+            if self.MEGA_SUFFIX:
+                self.cards[10 ** (n - 3)] = word + self.MEGA_SUFFIX
+                if self.cards[10 ** (n - 3)] == 'milião':
```

Comment:
This doesn't make any sense, if you are implementing `set_high_numwords` just set the suffixes you need, no need to use class props

Reply:
I just used the set_high_numwords from lang_EU but I needed to correct the "milião". 
Is it better if I correct it in merge? Like:
```python
if ntext == 'milião':
    ntext = 'milhão'
```